src/components/AlignGroup.tsx

```diff
@@ -10,6 +10,7 @@ export interface AlignGroupProps {
   verticalAlign?: Align;
   horizontalAlign?: Align;
   children: ReactNode | ReactNode[];
+  none?: boolean;
```

Comment:
Sorry if I wasn't clear, the idea was to add a 4th option to the `Align`, so verticalAlign and horizontalAlign can be `'none'` (which becomes the default value)

Reply:
Okey no problem